### ISSUE TITLE
Read the songs in succession from the "Titles" tab rather than just the selected one.

### ISSUE THREAD

**Issue by elyaB** [gateship-one/odyssey#174]
I don't know yet if this is a bug report or a feature proposal : 

When I go to the **"Titles"** tab (the third one) and select a song in the list, I usually expect it to start from this place and continue to read the following songs, instead of just putting the title I choose in the queue.

- Right now, it just plays one song.
- It needs to add all the following songs in the queue.

If I'm not clear, please tell me to edit :)

**Comment by gnome17** [comment]
Hi,

actually it is a feature proposal because it works as intended ;-)

We did this because it could be annoying to add everything if the user has a very large amount of music on his phone. But maybe we could add this as an option. We will think about it.

**Comment by elyaB** [comment]
Yes I think it would be a good option, and I also think this should be the default behavior. As a user, I expect it to be this way (as other players function this way)
It was quite surprising for me to notice that my song was playing in repeat when i launched it from the tiltes tab ; then I discovered only this song was added to the queue.
I don't think this is useful ? 
For example, what I used to do with other players is select a song from my titles tab, then enable the random mode. This way, I could choose to start my random playlist with the title I wanted.
Also, I used to listen to my song in alphabatical order a lot. With Odyssey, I can't with one click (I have to add all manualy, and I have a large amount of songs....)
In my opinion, an option that allows the user to play only one song (for large amount for example), and a default behavior that adds all the songs in the queue is the best choice.
That's only my opinion but in my experience, every player works this way :)

**Comment by rdy2go** [comment]
Since the last Comment on this Issue was in 2019 some things changed in the Software.

In Version 1.1.20 (release-32) there was added an Option to sort by Filename instead of Track Number.

With regard to that Design decision of the dev's to keep the "Tracks" Tab the way it is, the OP has now the possibility to use that new added function.

Maybe a reason to close that issue?
